server/etcdmain/help.go

```diff
@@ -257,6 +272,10 @@ Experimental feature:
     Skip verification of SAN field in client certificate for peer connections.
   --experimental-watch-progress-notify-interval '10m'
     Duration of periodical watch progress notification.
+  --experimental-downgrade-check-time
+    Duration of time between two downgrade status checks.
+  --experimental-memory-mlock
```

Comment:
We should add the default value for each flag as well. This is just a backporting PR, please feel free to resolve it in a separate PR.

THanks.